docs/lesson-plans/week-1.md

```diff
@@ -0,0 +1,720 @@
+special image files. They're put here for easy reference and access.
+
+src/ is the directory with your actual project files. All of your React components,
+views, tests, and code will be inside here.
+
+package.json is a file containing dependency/package information, scripts to be run using npm,
+and other data about your project.
+
+package-lock.json holds information on requested dependencies, which dependencies are required for 
+the requested dependencies, and special information that should rarely change. You should never have 
+to do anything with this file, and
+it should **_definitely_** be included in a Git repository.
+
+tsconfig.json holds information about your Typescript configuration.
+
+#### Backend
+
+pom.xml is a file with project build instructions, dependencies, and other project information.
+
+api/ is a directory handling routing information and data transfer objects (DTOs).
+
+persist/ is a directory for handling and saving data.
+
+service/ is the entry point for your project, so you'll start it up from there. It handles
+connecting everything together and basically 'running' your project.
+
+### What will each week look like?
+
+Every week, we'll have one of these meetings where we'll go over new concepts that
+you'll need to build your project. After the meetings, we may give out a description, starter
+code, and other resources for what we think you should work on during the upcoming week. 
+You're always free to work on other parts of your project or not do it at all if that's
+what you want to do. The following weekend before the next meeting, we'll send out
+what we're calling 'catch-up code,' which is our implementation of the concepts and
+descriptions we told you to work on during the week. This catch-up code will show
+the current state of the project and the progression of our app over time. The
+reason we're doing this is because we understand that there are weeks that you may 
+be sick, drowning in work, just not in the mood to work on your project, or stuck on some 
+issues, but you'll probably want to stay caught up with the material. This is our way of 
+ensuring everyone has the opportunity to be on the same page.
+
+### Important Links
+
+Just in case you don't have the links or dont't know where they are, here are some 
+important course resources.
+
+>Note: there is also a dictionary of common terms we've compiled for you which should be
+>available in the Jumpstart Docs
+
+[C4C Jumpstart Docs](https://learn.c4cneu.com) (where this should be)
+
+[Jumpstart Frontend Starter Code (not yet done)]()
+
+[Jumpstart Backend Starter Code](https://github.com/Code-4-Community/jumpstart-backend/tree/starter-code)
+
+[C4C Slack](https://c4cneu.slack.com/)
+
+[C4C GitHub Repository](https://github.com/Code-4-Community)
+
+[C4C Website](https://c4cneu.com)
+
+[C4C Dev Docs](https://docs.c4cneu.com)
+
+#### Making edits to the dictionary
```

Comment:
On the dictionary note, and following up to some other comments about Jumpstarter's inevitably encountering numerous errors throughout this process - is it worth it to setup a dictionary style list of common errors and diagnoses?

Reply:
Possibly? I'm not really sure which errors can occur, but I left a troubleshooting guide of some that I encountered while setting up somewhere in the doc. I think that would be really useful for future semesters once we get data on that.